packages/browser/src/Component/Itowns/3DTiles/3DTilesUtils.js

```diff
@@ -1,4 +1,145 @@
 import * as THREE from 'three';
+import { TilesManager } from './TilesManager';
+
+/**
+ * Create a geometric outliers for each tile of a tileset when its geometry is loaded
+ * Each outlier tile geometry does not keep the batchid for each outliers.
+ * May be slow to create / load.
+ *
+ * @param {TilesManager} tilesManager A TilesManage object from UD-Viz.
+ */
+export function createOutliersOfTileset(tilesManager) {
+  tilesManager.addEventListener(
+    TilesManager.EVENT_TILE_LOADED,
+    function (tile) {
+      if (tilesManager.tiles[tile.tileId] != undefined) {
+        if (tilesManager.tiles[tile.tileId].cityObjects != undefined) {
+          const geom =
+            tilesManager.tiles[tile.tileId].getObject3D().children[0]
+              .children[0];
+          if (!geom.hasOutlier) {
```

Comment:
when creating a wireframe object, tag it in userData with something like isWireframe, then to check if an object has already a wireframe attach to it do something like:
let hasWireframe = false;
object.traverse((child)=>{
if(!child.isWireframe)return;
hasWireframe = true;
})